nx/include/switch.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
 #include "switch/services/apm.h"
 #include "switch/services/applet.h"
 #include "switch/services/audout.h"
-#include "switch/services/bsd.h"
+//#include "switch/services/bsd.h" Use switch/runtime/devices/socket.h instead
```

Comment:
Why commented?

Reply:
It could be said that this is part of avoiding importing ugly BSD headers by default :p

Reply:
Fix should be to remove ugly-bsd-header-dependency on bsd-service shim